composeApp/src/commonMain/kotlin/org/dots/game/LocaleProvider.kt

```diff
@@ -0,0 +1,31 @@
+package org.dots.game
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.compositionLocalOf
+import java.util.Locale
```

Comment:
It makes the app noncompilable on platforms other than JVM. It would be great to have compilable app at least for WasmJS (at least a plug if WasmJS doesn't support the current localisation feature).